File: src/acceptance-test/java/net/slipp/tag/TagAT.java
package net.slipp.tag;

import net.slipp.support.AbstractATTest;

import org.junit.Before;
import org.junit.Test;

public class TagAT extends AbstractATTest {
    @Before
    public void setup() {
        super.setup();
        driver.get("http://localhost:8080");
    }
    
    @Test
    public void request_add_tag() throws Exception {
        loginToFacebook(1);
        RequestTagPage requestTagPage = indexPage.goRequestTagPage();
    }
    
    
}

File: src/acceptance-test/scala/net/slipp/domain/tag/TagAT.scala
package net.slipp.domain.tag

import net.slipp.support.AbstractATTest
import org.junit.Before
import org.junit.Test

class TagAT extends AbstractATTest {
  @Before override def setup {
    super.setup
    driver.get("http://localhost:8080")
  }

  @Test
  def request_add_tag {
    loginToFacebook(1)
    val requestTagPage: RequestTagPage = indexPage.goRequestTagPage
  }
}
